x-pack/plugins/observability_shared/common/locators/observability/slo_edit.ts

```diff
@@ -5,19 +5,39 @@
  * 2.0.
  */
 
-import { setStateToKbnUrl } from '@kbn/kibana-utils-plugin/public';
+import { setStateToKbnUrl } from '@kbn/kibana-utils-plugin/common';
+import { BudgetingMethod, Indicator, TimeWindow } from '@kbn/slo-schema';
 import type { RecursivePartial } from '@elastic/charts';
 import type { SerializableRecord } from '@kbn/utility-types';
 import type { LocatorDefinition } from '@kbn/share-plugin/public';
-import type { CreateSLOForm } from '../pages/slo_edit/types';
-import { sloEditLocatorID } from '../../common';
-import { SLO_CREATE_PATH, SLOS_PATH } from '../routes/paths';
+import { OBSERVABILITY_APP_BASE_PATH } from '../../constants';
+
+interface CreateSLOForm {
```

Comment:
This type needs to be duplicated from the Observability app, because Observability Shared may not depend on Observability. Only the other way around.